easybuild/easyconfigs/y/Yambo/Yambo-3.4.2-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'Yambo'
+version = '3.4.2'
+
+homepage = 'http://www.yambo-code.org'
+description = """Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.
+ Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit, and PWscf."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+dependencies = [('netCDF-Fortran', '4.4.2')]
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['http://qe-forge.org/gf/download/frsrelease/208/932/yambo-3.4.2.tgz']
```

Comment:
you should strip off the actual tarball name here, EB adds that already through `sources`